src/components/Heading/Heading.tsx

```diff
@@ -136,14 +157,29 @@ export const Heading = chakra(
       ) : (
         contentToRender
       );
-
+      /* The syntax for responsive styles is not working properly for fontSize
```

Comment:
This is the block of code that I mentioned during standup last week.  This handles the responsive font sizing for the `Heading` component.

Reply:
I'll look closer into this.